shacl/index.html

```diff
@@ -461,7 +461,7 @@
 	foaf:mbox &lt;mailto:bob@example.org&gt; ;
 	foaf:mbox &lt;mailto:rs@example.org&gt; .</pre>
 				<p>
-					The SHACL validation would validate <code>inst:Issue1</code> and <code>inst:Issue3</code> against <code>ex:IssueShape</code>
+					The SHACL validation would validate data graphs <code>inst:Issue1</code> and <code>inst:Issue3</code> against shapes graph<code>ex:IssueShape</code>
```

Comment:
I would change it to:
"The SHACL validation would validate <code>inst:Issue1</code> and <code>inst:Issue3</code> in the data graph against shapes graph<code>ex:IssueShape</code> in the shapes graph"
or leave it as is.
putting data graph and shapes graph in front of the resources is like the resources are the actual graphs while these resources only exist as subjects in triples in the respecting graphs